test/pytorch_test_base.py

```diff
@@ -464,7 +464,9 @@ def on_tpuvm():
 
 class XLATestBase(DeviceTypeTestBase):
   device_type = 'xla'
-  unsupported_dtypes = {torch.half, torch.complex64, torch.complex128}
+  unsupported_dtypes = {
```

Comment:
This got leaked during rebasing, would this be OK? cc @JackCaoG

Reply:
`torch.complex32` is no longer exist on the upstream, we must not add it back in pt/xla

Reply:
I see, so keep this?
```
unsupported_dtypes = { ... torch.complex32 ... }
```

Reply:
You can just revert the change in this file(by patching a new commit for example).

Reply:
Reverted the change.